src/jdataview.js

```diff
@@ -244,6 +238,71 @@ Int64.fromNumber = function (number) {
 	return new Int64(lo, hi);
 };
 
+
+// Precise versions of toString() for Int64/Uint64
+
+Uint64.prototype.toString = function() {
+	// Faster toString() for numbers which can be represented precisely as floats.
+	if (this.hi < pow2(19)) {
+		return Number.prototype.toString.apply(this.valueOf(), arguments);
+	}
+
+	// This converts the numbers to base 10 digit arrays for arbitrary precision toString().
+	// See http://www.danvk.org/hex2dec.html
+
+	// e.g. 1234 -> [4, 3, 2, 1]
+	var numToDigits = function(num) {
```

Comment:
This and `add` functions don't seem to depend on any scope variables and most engines are still stupid enough to recreate closures on each visit which is unwanted in our case. Could you please move them out (right before `toString`) as regular functions?